docs/index.yml

```diff
@@ -202,53 +196,57 @@ additionalContent:
           - url: /aspnet/core/blazor/
             text: "Blazor: Interactive client-side web UI with .NET"
       # Card
-      - title: Cloud native and microservices
-        links:
-          - url: ./azure/index.yml
-            text: Azure for .NET documentation
-          - url: /azure/storage
-            text: Storage
-          - url: ./fsharp/using-fsharp-on-azure/index.md
-            text: Using F# on Azure
-          - url: architecture/serverless/index.md
-            text: "Serverless apps: Architecture, patterns, and Azure implementation"
-          - url: architecture/microservices/index.md
-            text: ".NET Microservices: Architecture for Containerized .NET Applications"
-      # Card
       - title: Mobile
         links:
-          - url: /xamarin/ios/
+          - url: /xamarin/xamarin-forms
+            text: Xamarin.Forms
+          - url: /xamarin/ios
             text: Xamarin.iOS
           - url: /xamarin/android
             text: Xamarin.Android
-          - url: /xamarin/xamarin-forms
-            text: Xamarin.Forms
           - url: /azure/developer/mobile-apps
             text: Develop Xamarin apps with Azure
       # Card
       - title: Desktop
         links:
           - url: /uwp
             text: Universal Windows apps
-          - url: /dotnet/desktop/wpf/
-            text: Windows Presentation Foundation (.NET Core)
+          - url: /dotnet/desktop/wpf/?view=netdesktop-5.0
```

Comment:
Same comment about view here - will this link automatically go to netdesktop-6.0 once it's available?